File: ClientSide/src/app/common/models/extraSelectModel.ts
import { SelectModel } from './selectModel';

export class ExtraSelectModel {

    public id: number;
    public title: string;
    public options: Array<SelectModel>;

    constructor (
        id: number,
        title: string,
        options: Array<SelectModel>
    ) {
        this.id = id;
        this.title = title;
        this.options = options;
    }
}

File: ServerSide/SAMS.BusinessLogic/Models/Common/ExtraSelectModel.cs
using System.Collections.Generic;
using System.Linq;

namespace SAMS.BusinessLogic.Models.Common
{
    public class ExtraSelectModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public IEnumerable<SelectModel> Options { get; set; }
    }
}
